go/pkg/sysdb/coordinator/coordinator.go

```diff
@@ -107,8 +107,8 @@ func (s *Coordinator) GetCollection(ctx context.Context, collectionID types.Uniq
 	return s.catalog.GetCollection(ctx, collectionID, collectionName, tenantID, databaseName)
```

Comment:
we should clean this up, not sure why it was introduced in the first place. There is scope for consolidating these Get APIs